src/nni_manager/training_service/kubernetes/kubernetesTrainingService.ts

```diff
@@ -327,5 +329,34 @@ abstract class KubernetesTrainingService {
 
         return Promise.resolve();
     }
+
+    protected async createRegistrySecret(filePath: string | undefined): Promise<string> {
+        if(filePath === undefined || filePath === '') {
+            return Promise.resolve('');
```

Comment:
not necessary to wrap '',   just return '' will be cleaner.

Reply:
fixed